client/src/components/SearchReaslt/CharityCount/index.js

```diff
@@ -8,6 +8,25 @@ const CharityCount = props => {
   const { count, select } = props;
   let str = '/compare?';
   str += `${select.map((x, i) => `charity${i + 1}=${x}`).join('&')}`;
+  if (count < 2) {
+    return (
+      <div className="count-charity">
+        <div className="tooltip">
+          <img
+            className="img-charity"
+            src="https://image.flaticon.com/icons/svg/1170/1170651.svg"
+            alt="img charity"
+          />
+          <div className="count">
+            <strong>{count}</strong>
+          </div>
+          <span className="tooltiptext place">
+            Add and Compare at least <strong>two</strong>
+          </span>
+        </div>
+      </div>
+    );
+  }
```

Comment:
@Asmaa-ezz 
These two return cases repeat a lot of the same jsx.  I would suggest sharing the common jsx and moving the if-statement a little lower - it looks the only part that changes is the tooltiptext span, so you could move the conditional there.  Here's one way to do it:
```
{count < 2 && (
  <span clsasName="tooltiptext place">
    Add and compare at least <strong>two</strong>
  </span>
)}
{count >= 2 && (
  <span className="tooltiptext">Compare</span>
)}
```
Removing that duplication will make this code a little bit easier to read, and will be easier to change in the future (i.e. if you decided to change the img-charity src, you'd only have to change it in one place instead of two - there's no chance of forgetting one of the two places).